src/components/InstanceMonitoring/InstanceMonitoringCard.js

```diff
@@ -1,42 +1,38 @@
 import cx from 'classnames';
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Icon } from 'carbon-components-react';
 import { iconChevronUp } from 'carbon-icons';
 
-export default class InstanceMonitoringCard extends Component {
-  static propTypes = {
-    className: PropTypes.string,
-    hideSummaryText: PropTypes.node.isRequired,
-  };
+const InstanceMonitoringCard = ({
+  children,
+  className: customClassName,
+  hideSummaryText,
+}) => {
+  const className = cx('infra--instance-card', customClassName);
 
-  static defaultProps = {
-    hideSummaryText: 'Hide Summary',
-  };
+  return (
+    <section className={className}>
+      <ul className="infra--instance-metrics">{children}</ul>
+      <div className="infra--instance-hide">
+        <hr />
+        <span className="infra--instance-hide-link">
+          {hideSummaryText}
+          <Icon className="chevron-icon" icon={iconChevronUp} />
+        </span>
+        <hr />
+      </div>
+    </section>
+  );
+};
 
-  render() {
-    const {
-      children,
-      className: customClassName,
-      hideSummaryText,
-    } = this.props;
-    const className = cx({
-      'infra--instance-card': true,
-      [customClassName]: true,
-    });
+InstanceMonitoringCard.propTypes = {
+  className: PropTypes.string,
+  hideSummaryText: PropTypes.node.isRequired,
```

Comment:
Could you please add a short description of the props as comments here, as some components in `carbon-components-react` does? (Ref: https://github.com/IBM/carbon-components-react/pull/1195) Thanks!

Reply:
Hi @asudoh could you give me an example in the code as to how this would written out? Thanks

Reply:
HI @chrsfshr please refer to: 
https://github.com/carbon-design-system/carbon-addons-infrastructure/pull/29#discussion_r212782970 - Thanks!